dpnp/dpnp_iface_mathematical.py

```diff
@@ -253,6 +255,78 @@ def copysign(x1, x2, **kwargs):
     return call_origin(numpy.copysign, x1, x2, **kwargs)
 
 
+def cumprod(x, **kwargs):
+    """
+    Return the cumulative product of elements along a given axis.
+
+    For full documentation refer to :obj:`numpy.cumprod`.
+
+    Limitations
+    -----------
+        Parameter ``x`` is supported as :obj:`dpnp.ndarray`.
+        Keyword arguments ``kwargs`` are currently unsupported.
+        Otherwise the functions will be executed sequentially on CPU.
+        Input array data types are limited by supported DPNP :ref:`Data types`.
+
+    Examples
+    --------
+    >>> import dpnp as np
+    >>> a = np.array([1, 2, 3])
+    >>> result = np.cumprod(a)
+    >>> [x for x in result]
+    [1, 2, 6]
+    >>> b = np.array([[1, 2, 3], [4, 5, 6]])
+    >>> result = np.cumprod(b)
+    >>> [x for x in result]
+    [1, 2, 6, 24, 120, 720]
+
+    """
+
+    if not use_origin_backend(x) and not kwargs:
+        if not isinstance(x, dparray):
+            pass
+        else:
+            return dpnp_cumprod(x)
+
+    return call_origin(numpy.cumprod, x, **kwargs)
+
+
+def cumsum(x, **kwargs):
```

Comment:
please don't use single letter variable names. May be `x1` is a bit better than just an `x`